libs/coin-modules/coin-polkadot/src/logic/estimatedFees.ts

```diff
@@ -0,0 +1,24 @@
+import { getAbandonSeedAddress } from "@ledgerhq/cryptoassets";
+import { loadPolkadotCrypto } from "./polkadot-crypto";
+import { craftTransaction, defaultExtrinsicArg } from "./craftTransaction";
+import { fakeSignExtrinsic } from "./signTransaction";
+import polkadotAPI from "../network";
+
+export default async function estimatedFees({
```

Comment:
I didn't check the big picture but I find it weird for this method to estimateFees on a default transaction and not use currently crafted operation, maybe it works in our case here but for example on cosmos we have to estimateFees accurately and transaction would need to be an input of this method